packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import IsDarkContext from "../../context";
+import { MAXIMUM_NUM_OF_CHARACTER } from "../../constants";
+
+import * as sc from "./Link.styles";
+
+const truncate = str =>
+  str.length > MAXIMUM_NUM_OF_CHARACTER ? `${str.substring(0, MAXIMUM_NUM_OF_CHARACTER)}...` : str;
+
+function isString(item) {
+  return typeof item === "string" || item instanceof String;
+}
+
+function Link(props) {
+  const { children, href, as, ...moreProps } = props;
+  const shouldTruncate = isString(children) && children.length > MAXIMUM_NUM_OF_CHARACTER;
+
+  const link = (
+    <IsDarkContext.Consumer>
+      {isDark => (
+        <sc.Link data-pka-anchor="breadcrumbs.link" as={as} kind="minor" href={href} isDark={isDark} {...moreProps}>
+          {shouldTruncate ? truncate(children) : children}
+        </sc.Link>
+      )}
+    </IsDarkContext.Consumer>
+  );
+
+  return (
+    <sc.ListItem data-pka-anchor="breadcrumbs.listItem">
+      {shouldTruncate ? (
+        <Popover isDark isEager>
```

Comment:
Are you sure it's supposed to be dark? I haven't seen the designs, so I don't know, but if it's extra long than I think it's usually supposed to be white.

cc @sunminsarahlee

Reply:
Here is the design doc if you need: https://aclgrc.atlassian.net/wiki/spaces/UX/pages/1288077545/Breadcrumbs+approved

Reply:
Perfect!